firedrake/slate/slate.py

```diff
@@ -567,7 +568,7 @@ class Action(TensorOp):
     :arg function: a :class:`firedrake.Function` object.
     """
 
-    prec = 2
+    prec = 3
```

Comment:
All of the eigen tensor expressions are created as a `FlatBlock` (since they're method calls).  Hence they want parenthesising here.